src/styles/main.scss

```diff
@@ -1,3 +1,73 @@
-body {
+$calendar-border: 20px 10px;
+$calendar-gaps: 1px;
+$day-size: 100px;
+$day-background: #eee;
+$day-background-hover: pink;
+$day-border-color: #000;
+$day-font: arial, helvetica, sans-serif;
+
+$day-of-week:
+  "mon" 0,
+  "tue" 1,
+  "wed" 2,
+  "thu" 3,
+  "fri" 4,
+  "sat" 5,
+  "sun" 6;
+
+body,
+p {
   margin: 0;
 }
+
+.calendar {
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  gap: #{$calendar-gaps};
+  padding: #{$calendar-border};
+  width: 750px;
+
+  &__date {
+    box-sizing: border-box;
+    width: #{$day-size};
+    height: #{$day-size};
+
+    text-align: center;
+    line-height: 100px;
+    background-color: #{$day-background};
+    border: 1px solid #{$day-border-color};
+
+    cursor: pointer;
+    transition: transform 0.5s;
```

Comment:
What about background-color?
Also, let's create a variable for transition-duration as it should be the same per page

Reply:
You change background-color on hover, but don't include it in transition